lib/logs.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:logs/src/logging_service.dart';
+
+/// Enable (or disable) logging for all events on the given [channel].
+void enableLogging(String channel, [bool enable = true]) {
```

Comment:
Do we need these top level if we have a top-level `LoggingService` instance?

Reply:
My thinking is that the logging service will stay package-private, limiting the public API to a small collection of functions.  Having them top-level is in the spirit of flutter's `debug*` APIs. This is despite my general leaning towards objects --- my first cut in https://github.com/flutter/flutter/pull/21505 hung logging calls off of `Logger` objects...  My gut says this will be easier to land in flutter if we ever decide to inline any of this in foundation.  Up for whatever though!

Reply:
Yeah, lots of ways to slice it. Perhaps the service protocol hookup, and use in sample code, would help suss out what API style works best.